js/pulse.js

```diff
@@ -197,12 +198,18 @@ var PulseMaker = {
   'toggleAdditionalFeatures' : function () {
     if (FEATURE.orphans) { typography.preventTextOrphans(); }
   },
+  generatePulseProjectId(projectData) {
+    var timestamp = projectData.Timestamp ? projectData.Timestamp : false;
+    var projectId = timestamp ? Date.parse(timestamp) : '0';
```

Comment:
I know this was just copied from above, but I wanted to raise a couple things:

* Will all projects have a unique time stamp? What about batch import projects (we might just migrate all hive portfolio projects at once)?
* More of a design suggestion, but this isn't a very readable in the url. Can we use a slug version of the title + maybe append a timestamp? Something like what Medium does?

Neither point is a huge deal now, especially if the url structure will change down the road, but something I'd like to keep in mind.